docs/changelogs/v0.37.md

```diff
@@ -0,0 +1,35 @@
+# Kubo changelog v0.37
+
+<a href="https://ipshipyard.com/"><img align="right" src="https://github.com/user-attachments/assets/39ed3504-bb71-47f6-9bf8-cb9a1698f272" /></a>
+
+This release  was brought to you by the [Interplanetary Shipyard](https://ipshipyard.com/) team.
+
+- [v0.37.0](#v0370)
+
+## v0.37.0
+
+- [Overview](#overview)
+- [🔦 Highlights](#-highlights)
+  - [Clear provide queue when reprovide strategy changes](#clear-provide-queue)
+  - [📦️ Important dependency updates](#-important-dependency-updates)
+- [📝 Changelog](#-changelog)
+- [👨‍👩‍👧‍👦 Contributors](#-contributors)
+
+### Overview
+
+### 🔦 Highlights
+
+#### Clear Provide Queue
+    
+On restart, if the currently configured reprovide strategy does not match the previous strategy read from the datastore, then clear the provide queue and update the reprovide strategy that is stored in the datastore.
+
+> [!NOTE]
+> Any CIDs queued at shutdown won’t be provided until the next reprovide ([`Reprovider.Interval`](../config.md#reproviderinterval) after restart), even if they match the new reprovide strategy.
```

Comment:
nit: we need to use absolute URLs because this amrkdown is also used on https://github.com/ipfs/kubo/releases
```suggestion
> Any CIDs queued at shutdown won’t be provided until the next reprovide ([`Reprovider.Interval`](https://github.com/ipfs/kubo/blob/master/docs/config.md#reproviderinterval) after restart), even if they match the new reprovide strategy.
```